Welolo/cypress/integration/InputField.js

```diff
@@ -0,0 +1,20 @@
+class InputField{
+
```

Comment:
I would recommend possibly adding the steps you put in the comment into the readme for easy future access :)